client/src/pages/teds/charts/PrctIpbesPublicationsByCountry.tsx

```diff
@@ -0,0 +1,60 @@
+import Highcharts from 'highcharts';
+import HighchartsReact from 'highcharts-react-official';
+import buildSeries from './utils/UseSeries.js';
+import QueryResponse from './utils/UseQueryResponse.js'
+
+
+
+export default function PrctIpbesPublicationsByCountry() {
+
+  const body = {
+    should: [
+      { term: { 'ipbes.chapter.keyword': '1' } },
+      { term: { 'ipbes.chapter.keyword': '2.1' } },
+      { term: { 'ipbes.chapter.keyword': '2.2' } },
+      { term: { 'ipbes.chapter.keyword': '2.3' } },
+      { term: { 'ipbes.chapter.keyword': '3' } },
+      { term: { 'ipbes.chapter.keyword': '4' } },
+      { term: { 'ipbes.chapter.keyword': '5' } },
+      { term: { 'ipbes.chapter.keyword': '6' } },
+      { term: { 'ipbes.chapter.keyword': 'glossary' } },
+    ],
+    minimum_should_match: 1,
+  };
+
+  const {data, isLoading} = QueryResponse(body, 20, 'ipbes')
+
+  if (isLoading) {
+    return <div>Loading...</div>;
+  }
+
+  const {series, categories}= buildSeries(data)
+
+
+  const options = {
```

Comment:
Could it be factorized ? Then the specific fields like "title" should be overloaded.